src/style.css

```diff
@@ -1,3 +1,73 @@
+:root {
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider specifying fallback font-families in case 'Roboto' is not available. For instance, 'font-family: Roboto, Arial, sans-serif;' ensures that if 'Roboto' fails to load, 'Arial' will be used, and if both custom fonts fail, the browser's default sans-serif font will be used.